pkg/skaffold/event/v2/application_logs.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2021 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v2
+
+import proto "github.com/GoogleContainerTools/skaffold/proto/v2"
+
+func ApplicationLog(podName, containerName, message string) {
```

Comment:
I forgot if application logs have dev-iteration id in it.  Do you remember if we need to preserve the iteration id?

Reply:
the original design doesn't have them, but I think we definitely should make the change to include them to make things easier for the CC team. I can open a todo issue for that and do it in a follow up PR to avoid cluttering this one

Reply:
https://github.com/GoogleContainerTools/skaffold/issues/5837